vars/runIntegTestScript.groovy

```diff
@@ -25,14 +31,31 @@ void call(Map args = [:]) {
     String component = args.componentName
     echo "Component: ${component}"
 
-    sh([
+    String switchUser = args.switchUserNonRoot ?: 'false'
+    echo "Switch User to Non-Root (uid=1000): ${switchUser}"
+
+    String currentDir = sh (
+        script: 'pwd',
+        returnStdout: true
+    ).trim()
+    String switchCommandStart = switchUser.equals('true') ? "su - `id -un 1000` -c \" cd ${currentDir} &&" : ''
```

Comment:
We could maybe use groovy `toBoolean()` for the conditional check here.

Reply:
Hi @zelinh toBoolean() can ignore a lot of noise such as `t`, `1`, `TRue`.
I will add a check to make sure the arg pass only `true` or `false`.

Thanks.